tests/v2/validation/provisioning/rke2/README.md

```diff
@@ -0,0 +1,202 @@
+
+# RKE2 Provisioning Configs
+
+For your config, you will need everything in the Prerequisites section on the previous readme, [Define your test](#provisioning-input), and at least one [Cloud Credential](#cloud-credentials) and [Node Driver Machine Config](#machine-rke2-config) or [Custom Cluster Template](#custom-cluster), which should match what you have specified in `provisioningInput`. 
+
+Your GO test_package should be set to `provisioning/rke2`.
+Please see below for more details for your config. 
+
+1. [Prerequisites](../README.md)
+2. [Define your test](#provisioning-input)
+3. [Cloud Credential](#cloud-credentials)
+4. [Configure providers to use for Node Driver Clusters](#machine-rke2-config)
+5. [Configuring Custom Clusters](#custom-cluster)
+6. [Back to general provisioning](../README.md)
+
+## Provisioning Input
+provisioningInput is needed to the run the RKE2 tests, specifically kubernetesVersion, cni, and providers. nodesAndRoles is only needed for the TestProvisioningDynamicInput test, node pools are divided by "{nodepool},". 
+
+**nodeProviders is only needed for custom cluster tests; the framework only supports custom clusters through aws/ec2 instances.**
+
+```json
+"provisioningInput": {
+    "nodesAndRoles": [
+      {
+        "etcd": true,
+        "controlplane": true,
+        "worker": true,
+        "quantity": 1,
+      },
+      {
+        "worker": true,
+        "quantity": 1,
+      }
+    ],
+    "kubernetesVersion": ["v1.21.6+rke2r1"],
+    "cni": ["calico"],
+    "providers": ["linode", "aws", "do", "harvester"],
+    "nodeProviders": ["ec2"]
+  }
+```
+
+## Cloud Credentials
+These are the inputs needed for the different node provider cloud credentials, inlcuding linode, aws, digital ocean, harvester, azure, and google.
+
+### Digital Ocean
+```json
+"digitalOceanCredentials": {
+   "accessToken": ""
+  },
+```
+### Linode
+```json
+"linodeCredentials": {
+   "token": ""
+  },
+```
+### Azure
+```json
+"azureCredentials": {
+   "clientId": "",
+   "clientSecret": "",
+     "subscriptionId": "",
+     "environment": "AzurePublicCloud"
+  },
+```
+### AWS
+```json
+"awsCredentials": {
+   "secretKey": "",
+   "accessKey": "",
+   "defaultRegion": ""
+  },
+```
+### Harvester
+```json
+"harvesterCredentials": {
+   "clusterId": "",
+   "clusterType": "",
+   "kubeconfigContent": ""
+},
+```
+### Google
+```json
+"googleCredentials": {
+    "authEncodedJson": ""
+}
+```
+
+## Machine RKE2 Config
+Machine RKE2 pool config is the last thing need for the RKE2 provisioning tests
```

Comment:
This should be `Machine RKE2 config is the last thing needed for the RKE2 provisioning tests.`